FlowCal/excel_ui.py

```diff
@@ -470,6 +470,18 @@ def process_beads_table(beads_table,
                     mef = [int(e) if e.strip().isdigit() else np.nan
                            for e in mef]
                     mef_values.append(mef)
+
+            # Ensure matching number of `mef_values` for all channels (this
+            # implies that the calibration beads have the same number of
+            # subpopulations for all channels).
+            if mef_values:
+                if not np.all([len(mef_values_channel)==len(mef_values[0])
+                               for mef_values_channel in mef_values]):
+                    raise ExcelUIException("Must specify the same number of"
+                                           + " MEF Values for each channel."
+                                           + " Use 'None' to instruct FlowCal"
+                                           + " to ignore a detected"
+                                           + " subpopulation.")
```

Comment:
> We agreed that exception messages should start with lowercase; however some ExcelUIExceptions in this module start with lowercase, and some with uppercase.

true

> I guess uppercase is nice because this ends in a message in the Excel file.

agree

If you want to standardize this systematically across the entire module, I think that should be a separate issue (trivial as it is). I'm gonna assume you don't want any changes to this Pull Request, though.